scripts/fig2.py

```diff
@@ -2,10 +2,10 @@
 from common import label
 
 Figure(
-    "23.1cm", "6cm",
+    "28cm", "6cm",
     Panel(SVG(snakemake.input.dag).scale(0.24), label("a")),
     Panel(SVG(snakemake.input.workflow).scale(0.45), label("b")).move(220, 0),
     Panel(SVG(snakemake.input.comp), label("c")).move(310, 0),
-    Panel(SVG(snakemake.input.turnaround), label("d")).move(595, 0),
+    Panel(SVG(snakemake.input.turnaround), label("d")).move(595, 50),
```

Comment:
Why moving this down?

Reply:
The new aspect ratio is long and thin so I thought it looked better close-to-centered, otherwise there was a lot of whitespace below it. Easy enough to move back if you'd like.